node-install.sh

```diff
@@ -1,6 +1,71 @@
 #!/bin/bash
 
-echo 'Node.js Linux Installer by github.com/taaem, updated by github.com/MichaIng'
+echo 'Universal Node.js Linux Installer by github.com/taaem, updated by github.com/MichaIng'
+
+show_help() {
+    cat <<END
+
+Use this script to install the latest official version available for your CPU architecture.
+Alternatively install a specific unofficial build for rare architectures, using the parameters below.
+
+Usage: node-install.sh [<params>]
+
+Parameters:
+    -h,  --help                     Show this help screen
+    -lu, --list-unofficial          List all available versions in nodejs.org's unofficial builds
+    -u,  --unofficial [<version>]   Install either latest version or <version> from nodejs.org's
+                                    unofficial builds at unofficial-builds.nodejs.org/download/release/.
+                                    Syntax of <version> shall be e.g. "v15.5.1".
+END
+}
+
+ARCH=$(uname -m)
+UNOFFICIALS_URI=https://unofficial-builds.nodejs.org/download/release
+
+list_available() {
+    if command -v 'curl' > /dev/null && command -v 'sort' > /dev/null; then
+        if command -v 'tail' > /dev/null && command -v 'cut' > /dev/null; then
+            curl -sSf $UNOFFICIALS_URI/index.tab | grep $ARCH | cut -f1 | tail -n +2 | sort -Vr
+        # elif command -v 'jq' > /dev/null; then
+        #     curl -sSf "$UNOFFICIALS_URI/index.json" | jq -r '.[].version' | sort -Vr
+        else
+            # echo 'ERROR: Required command line tools tail + cut, OR jq not found.' >&2
+            echo 'ERROR: Required command line tools tail + cut not found.' >&2
+            exit 1
+        fi
+    else
+        echo 'ERROR: Required command line tools curl, sort not found.' >&2
+        exit 1
+    fi
+}
+
+# getting options
+while [[ $1 ]]; do
+    case $1 in
+    -h | --help )
+        show_help
+        exit
+        ;;
+    -lu | --list-unofficial )
+        list_available
+        exit
+        ;;
+    -u | --unofficial )
+        shift; unofficial_build_version=$1
+        if [[ -z $unofficial_build_version ]]; then
+            unofficial_build_version=$(curl -sSf $UNOFFICIALS_URI/index.tab | grep $ARCH | cut -f1 | tail -n +2 | sort -V | tail -n 1)
```

Comment:
Ah, as we sort out lines without matching architecture first (which includes the column names), `tail -n +2` removes that latest version.